### PR TITLE
feat(components): Loading

### PR DESCRIPTION
**Pull request by Guosugaz**

### CODE REVIEW

**Comment by IlhamTahir** [comment]
is it ready? @Guosugaz

**Comment by IlhamTahir** [comment]
@Guosugaz  if WIP, please add  `WIP:` at your pull request title

**Comment by Guosugaz** [comment]
> is it ready? @Guosugaz

it is ready

File: src/styles/animation.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@keyframes van-rotate {
```
Comment (mxdi9i7) [inline src/styles/animation.scss:1]: should be vant-rotate?

**Comment by mxdi9i7** [comment]
Could you also group the storybook examples into different storybook links? It's a little messy to have them all in one link

File: src/styles/animation.scss (hunk 1 above)
Comment (Guosugaz) [reply src/styles/animation.scss]: yes

**Pull request merged**